src/Ledgers.jl

```diff
@@ -18,7 +18,8 @@ import Instruments: instrument, symbol, amount, name, currency
 export Credit, Debit, Account, Ledger, Entry, AccountId, AccountInfo
 export id, balance, credit!, debit!, post!, instrument, symbol, amount, name, currency
 
-struct AccountId; value::UUID; end
+abstract type Identifier end
+struct AccountId <: Identifier; value::UUID; end
```

Comment:
I would always have this on 3 lines, also add a AccountId() constructor, instead of using AccountId(uuid4()).

Reply:
Sure. Good idea 👍